packages/frontend-api/src/Resources/config/graphql-types/MoneyDecorator.types.yml

```diff
@@ -0,0 +1,7 @@
+MoneyDecorator:
+    type: custom-scalar
+    config:
+        description: "Represents and encapsulates monetary value"
+        serialize: ["Shopsys\\FrontendApiBundle\\Model\\Money\\MoneyType", "serialize"]
+        parseValue: ["Shopsys\\FrontendApiBundle\\Model\\Money\\MoneyType", "parseValue"]
+        parseLiteral: ["Shopsys\\FrontendApiBundle\\Model\\Money\\MoneyType", "parseLiteral"]
```

Comment:
missing empty line at the end

Reply:
Fixed https://github.com/shopsys/shopsys/commit/4baacbc382f1b3e5f6ba3028a53e2784b29d253f